app/templates/base.html

```diff
@@ -30,7 +30,7 @@
     'tag': {
       'text': config['SERVICE_PHASE'],
     },
-    'html': 'This is a new service. Help us improve it and <a class="govuk-link" href="#">give your feedback by email</a>.'
+    'html': 'This is a new service. Send your feedback to <a class="govuk-link" href="#">mapd@justice.gov.uk</a> to help us improve it.'
```

Comment:
if you make this a mailto link then it will open the users default email application:
`<a href="mailto:mapd@justice.gov.uk">mapd@justice.gov.uk</a>`